de.fu_berlin.inf.dpp.stf.test/sonar-project.properties

```diff
@@ -0,0 +1,16 @@
+sonar.projectName        = Saros/Eclipse
```

Comment:
this file is obsolete. The sonar properties are maintained in the build.gradle (e.g. [here](https://github.com/saros-project/saros/blob/master/de.fu_berlin.inf.dpp.core/build.gradle)).
Add the following block to the build.gradle file of the project:
```gradle
sonarqube {
  properties {
    property 'sonar.projectName', <ADD NAME HERE>
    property 'sonar.projectDescription', <ADD DESCRIPTION HERE>
    property 'sonar.projectVersion', <ADD VERSION HERE>
  }
}
```